ui-spacetimechart/src/styles/main.css

```diff
@@ -1,3 +1,48 @@
 @import 'tailwindcss/base';
 @import 'tailwindcss/components';
 @import 'tailwindcss/utilities';
+
+.spacetimechart-tooltip {
+  position: absolute;
+  background-color: rgba(0, 0, 0, 0.85);
+  box-shadow: 0 4px 6px -2px rgba(0, 0, 0, 0.28);
+  color: white;
+  font-size: 0.875rem;
+  line-height: 20px;
+  border-radius: 10px;
+  padding: 6px 12px 7px 12px;
+  margin: 12px;
```

Comment:
Why did you add this margin ?

Reply:
It's used to offset the tooltip from the cursor. Without it, the tooltip's top-left corner would appear directly under the cursor.